Source/WebCore/css/fullscreen.css

```diff
@@ -54,6 +54,17 @@ video:fullscreen {
     -webkit-cursor-visibility: auto-hide;
 }
 
+#if PLATFORM(APPLETV)
```

Comment:
In `html.css`, we have `#if defined(WTF_PLATFORM_IOS_FAMILY) && WTF_PLATFORM_IOS_FAMILY`.

Should this be `#if defined(WTF_PLATFORM_APPLETV) && WTF_PLATFORM_APPLETV`?